modular_transformer/classical_mc.py

```diff
@@ -0,0 +1,122 @@
+from typing import Callable, Optional, Union
+
+import torch
+from torch import Tensor
+from torch.nn import ReLU, Softmax
+
+from .base import Transformer
+from .layers import ClassicalTransformerDecoderLayer, ClassicalTransformerEncoderLayer
+from .layers.attention_modules.attention_mechanisms.masking import AttentionMatrixMask
+from .layers.attention_modules.output_modules import LinearMCDOutputModule
+
+
+class ClassicalMCDTransformer(Transformer):
```

Comment:
Why is this not inheriting from ClassicalTransformer?

Reply:
Fair question, I will change it so that it does.